config/migrations/2024/20240705152938-add-mock-VZW-account.sparql

```diff
@@ -17,14 +17,16 @@ PREFIX regorg: <http://www.w3.org/ns/regorg#>
 PREFIX act: <http://data.lblod.info/id/concept/activiteit/>
 PREFIX pav: <http://purl.org/pav/>
 PREFIX org: <http://www.w3.org/ns/org#>
+PREFIX besluit: <http://data.vlaanderen.be/ns/besluit#>
 
-# MOCK VZW 
+# MOCK VZW
 INSERT DATA {
   GRAPH <http://mu.semte.ch/graphs/public> {
     <http://data.lblod.info/id/organisaties/3e5edbbb-26e0-49f9-9082-55f2297bd73a> a org:Organization ;
         mu:uuid '3e5edbbb-26e0-49f9-9082-55f2297bd73a';
         skos:prefLabel "Mock VZW" ;
         dc:description "Mock VZW for testing" ;
+        besluit:classificatie <http://data.lblod.info/id/concept/c6157470-9fb3-4e8d-a9b7-8737dbfa3642> ;
```

Comment:
move this to `org:classification <http://data.lblod.info/id/concept/c6157470-9fb3-4e8d-a9b7-8737dbfa3642> `